File: Source/Core/Util/FileSystem/DataUnit.cs
namespace Dodoco.Core.Util.FileSystem {

    public static class DataUnit {

        public static readonly double BYTE = 1.0D;
        public static readonly double KILOBYTE = 1000.0D * BYTE;
        public static readonly double MEGABYTE = Convert.ToUInt64(Math.Pow(KILOBYTE, 2.0D));
        public static readonly double GIGABYTE = Convert.ToUInt64(Math.Pow(KILOBYTE, 3.0D));
        public static readonly double TERABYTE = Convert.ToUInt64(Math.Pow(KILOBYTE, 4.0D));

    }

}

File: Dodoco.Svelte/Source/Util/DataUnit.ts
export class DataUnit {

    public static readonly BYTE: number = 1.0;
    public static readonly KILOBYTE: number = 1000.0 * DataUnit.BYTE;
    public static readonly MEGABYTE: number = Math.pow(DataUnit.KILOBYTE, 2.0);
    public static readonly GIGABYTE: number = Math.pow(DataUnit.KILOBYTE, 3.0);
    public static readonly TERABYTE: number = Math.pow(DataUnit.KILOBYTE, 4.0);

}